### ISSUE TITLE
Add `at-rule-descriptor-value-no-unknown`

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#8149]
### What is the problem you're trying to solve?

A dedicated rule to flag unknown values for descriptors within at-rules.

For example, `bar` is not a known value for the `syntax` descriptor for `@property`:

```css
@property --foo {
  syntax: bar;
}
```

This functionality is currently rolled into [`declaration-property-value-no-unknown`](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/), which should only target properties rather than descriptors too.

### What solution would you like to see?

Suggested blueprint:

- Name: `at-rule-descriptor-value-no-unknown`
- Description: Disallow unknown descriptors for at-rules.
- Message: `Unexpected unknown value "${value}" for descriptor "${descriptor}"`
- Primary option: `true`
- Secondary options: n/a
- Category: Avoid errors > Unknown
- Included in sharable configs?: Yes, recommended.
- Expanded description:

This rule considers values for properties defined within the CSS specifications to be known.

**Comment by jeddy3** [comment]
> Is the not to include secondary options, like in `declaration-property-value-no-unknown`, made with the future addition of languageOptions in mind?

Yes, exactly.

**Comment by ybiquitous** [comment]
Overall LGTM. 👍🏼 

I would like to suggest one minor thing. Can we clarify the suggested description as follows?

```diff
-Description: Disallow unknown descriptors for at-rules.
+Description: Disallow unknown values for descriptors within at-rules.
```

See also the `declaration-property-value-no-unknown` description:

https://github.com/stylelint/stylelint/blob/b2c99cef764643f3bd9539b34cdec58af882db88/lib/rules/declaration-property-value-no-unknown/README.md?plain=1#L3

**Comment by jeddy3** [comment]
Good catch. I've updated the blueprint.

**Comment by jeddy3** [comment]
I've renamed this `invalid`  to match https://github.com/stylelint/stylelint/issues/8152.
